sys/include/net/ieee802154.h

```diff
@@ -112,28 +112,43 @@ extern const uint8_t ieee802154_addr_bcast[IEEE802154_ADDR_BCAST_LEN];
 /** @} */
 
 /**
+ * @defgroup net_ieee802154_conf     IEEE802.15.4 compile configurations
```

Comment:
Very nit-picky, but the alignment is off by one now.

Reply:
alignment with which block?

Reply:
The 4-tab spacing.

Reply:
Should be 
```suggestion
 * @defgroup net_ieee802154_conf    IEEE802.15.4 compile configurations
```